GDJS/Runtime/events-tools/layertools.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * GDevelop JS Platform
+ * Copyright 2013-2016 Florian Rival (Florian.Rival@gmail.com). All rights reserved.
+ * This project is released under the MIT License.
+ */
+namespace gdjs {
+  export namespace evtTools {
+    export namespace layer {
```

Comment:
Sorry about this, actually, there is already a lot of things in cameratools.ts that are not related to camera so maybe it should just be there.